### PR TITLE
Update the repository badges - Closes #4235

### LINKED ISSUE #4235: Review and update repository badges
### Description
To benefit the most out of the repository README, we'd should
- Review and fix the out dated badges
- Find and add new useful badges

### Motivation
Keep project general information up to date and valid.

### PR DESCRIPTION
**Pull request by vibratious**
### What was the problem?

This PR resolves #4235

### How was it solved?
- Added PR welcome badge
- Added Discord chat badge
- Added Issues badge

### How was it tested?
visually

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
 ![GitHub package.json version](https://img.shields.io/github/package-json/v/LiskHQ/lisk-desktop)
 [![DeepScan grade](https://deepscan.io/api/teams/6759/projects/8871/branches/113511/badge/grade.svg)](https://deepscan.io/dashboard#view=project&tid=6759&pid=8871&bid=113511)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://makeapullrequest.com)
+![Discord](https://img.shields.io/discord/405002561775599619/606112254789681163?style=flat-square)
```
Comment (ManuGowda) [inline README.md:9]: @vibratious seems like discord link is broken?

**Review by ManuGowda — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vibratious) [reply README.md]: fixed

**Review by gadjacobs — CHANGES_REQUESTED** [review]
@vibratious thanks for the help. Updating the the node version should maybe be in another ticket

**Comment by vibratious** [comment]
@gadjacobs  Thanks for bring it up, it would be better to do that.

**Review by reyraa — APPROVED** [review]
Good job @vibratious

**Review by ikem-legend — APPROVED** [review]

**Review by gadjacobs — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

**Comment by ManuGowda** [comment]
@vibratious we are ready to merge the PR, however, the last commit is not signed. Could you sign the last commit 🙏🏼

**Pull request merged**